Cocoa/Categories/NGSCategories.m

```diff
@@ -274,8 +274,6 @@ - (NSGradient *)gradientWithAlpha:(CGFloat)alpha
 @implementation NSOpenGLContext (CGLContextAccess)
 - (CGLContextObj)cglContext;
```

Comment:
This whole function is redundant if the app won't compile on 10.2 (which it doesn't any more) — just delete it